implementations/micrometer-registry-prometheus/src/main/java/io/micrometer/prometheus/MicrometerCollector.java

```diff
@@ -14,17 +14,24 @@
  * limitations under the License.
  */
 package io.micrometer.prometheus;
+import static java.util.stream.Collectors.toList;
```

Comment:
Can you revert this?

Reply:
Yes i will do that as soon as i come back to my office.